sdk/src/signature.rs

```diff
@@ -93,6 +96,31 @@ impl From<Signature> for [u8; 64] {
     }
 }
 
+impl From<[u8; SIGNATURE_BYTES]> for Signature {
+    #[inline]
+    fn from(signature: [u8; SIGNATURE_BYTES]) -> Self {
```

Comment:
i guess it'd be a mess to just not implement an infallible constructor?  not sure if panicking in here rather than blindly accepting any proper length byte array is better :thinking:

Reply:
The library itself does the same:
https://docs.rs/ed25519/1.0.2/src/ed25519/lib.rs.html#249-253

Reply:
derp. why even have types? :upside_down_face: